ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -231,7 +231,7 @@ import '../nuxeo-button-styles.js';
             <nuxeo-data-table-row header>
               <nuxeo-data-table-checkbox
                 style$="[[_computeSelectAllVisibility(selectionEnabled, selectAllEnabled, multiSelection)]]"
-                checked="[[selectAllActive]]"
+                checked="[[_isExcludedItemsNotEmpty]]"
```

Comment:
Perhaps we could do a simpler implementation, using a computed binding, with `selectAllActive` and `_excludedItems` dependencies? E.g.:
```suggestion
                checked="[[_isChecked(selectAllActive, _excludedItems)]]"
```
and then the `_isChecked` could do all the logics you do in `_onCheckBoxTap` and `_toggleSelectAll`?

Also, have you verified if the current solution fixes the issue for the other listings (e.g., data grid, data list)?

Reply:
@Gabez0r checked for data grid.

Reply:
let me do the test for data list.